test/fixtures/index.js

```diff
@@ -16,7 +16,7 @@ var path = require('path')
       return path.join(BUILD_DIR, 'apps', name);
     }
   , bundledFile = function (name) {
-      return path.join(BUILD_DIR, name, 'bundle.min.js');
+      return path.join(BUILD_DIR, 'apps', name, 'bundle.min.js');
```

Comment:
Oh,
This was wrong and I fixed it. Apparently my tests are the only ones consuming that method. This edit is in it's own commit (in case you didn't want the PR I was going to recommend cherrypicking at least this commit).